### PR TITLE
Fix issue with variable type

### PR DESCRIPTION
**Pull request by WitoldAtSmartframe**
Update Downloader test

### CODE REVIEW

File: test/DownloaderTest.php [hunk 1]
```diff
@@ -17,25 +17,23 @@ class DownloaderTest extends TestCase
 
     public function testDownload(): void
     {
-        $this->markTestSkipped('Skipped until adjust download with new flow.');
+        if (extension_loaded('zip') === false) {
```
Comment (WitoldAtSmartframe) [inline test/DownloaderTest.php]: remove this condition

File: src/Downloader.php [hunk 2]
```diff
@@ -72,6 +74,5 @@ private function unzip(string $path, array $package): void
             throw new RuntimeException(sprintf('Cannot extract file "%s" from ZIPArchive located in %s', $package['file'], $filePath));
         }
         $zip->close();
-        unlink($filePath);
```
Comment (WitoldAtSmartframe) [inline src/Downloader.php]: undo this change

Comment (rollo-smartframe) [inline src/Downloader.php]: It is required here to unlink the file.

File: test/DownloaderTest.php (hunk 1 above)
Comment (rollo-smartframe) [inline test/DownloaderTest.php]: no needed check, ext-zip is required in composer
